.github/chronographer.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+branch-protection-check-name: Changelog entry
+action-hints:
+  check-title-prefix: "Chronographer: "
+  external-docs-url: >-
+    https://docs.pytest.org/en/latest/contributing.html#preparing-pull-requests
+  inline-markdown: >-
+    See
+    https://docs.pytest.org/en/latest/contributing.html#preparing-pull-requests
+    for details.
+enforce-name:
+  suffix: .rst
+# labels:
+#   skip-changelog: skip news
```

Comment:
@RonnyPfannschmidt plz create a `bot:chronographer:skip` label. Alternatively, we can uncomment this part and use a nicer name. `skip news` is what pip uses.

Reply:
https://github.com/pytest-dev/pytest/labels/skip%20news was created

Reply:
I've uncommented the config section. Marking the thread as resolved.